js/controllers/fetchMovies_controller.js

```diff
@@ -0,0 +1,17 @@
+mainApp.controller('fetchMoviesCtrl',
+ function ($scope,message,getMovieName_service) {
+        'ngInject';
+    $scope.apiData = message;
+    $scope.getMovieName = function (event) {
```

Comment:
remove getMovieName.
change apiData -> movieDetais use this in template movie in